hieradata/node/lsstcam-ccobtb.cp.lsst.org.yaml

```diff
@@ -0,0 +1,60 @@
+---
+nm::connections:
+  eno1:
+    content:
+      connection:
+        id: "eno1"
+        uuid: "03da7500-2101-c722-2438-d0d006c28c73"
+        type: "ethernet"
+        interface-name: "eno1"
+      ethernet: {}
+      ipv4:
+        address1: "139.229.175.83/26,139.229.175.126"
```

Comment:
![image](https://github.com/lsst-it/lsst-control/assets/948901/4f3c4b96-bdbf-4bd7-ae37-873ae21bff0c)
Missing device association.

Reply:
Device is associated on IPAM cp.